logistics/handling-money.md

```diff
@@ -0,0 +1,23 @@
+# Handling Money
+
+RustBridge should be free for attendees.
+It is possible to run an event on a very small budget.
+The two most common things that cost money are renting space and providing food for attendees.
+
+Approach local companies or universities for sponsorship.
+They might give you a venue for free or for a very small amount of money.
+If you can get a venue sponsor you already saved yourself what could be your biggest cost. 
+
+Food for attendees is another big cost. Try to find a sponsor to cover that.
+Remember, sponsors get publicity for their businesses or organizations out of sponsorship – which can mean an awful lot to them! But if all else fails, if you’ve got a space but no lunch, ask your participants to brown bag it.
```

Comment:
you might want to say "bring their own lunch" to avoid the confusing idiom. additionally, you can also make a list of local restaurants and supermarkets that are nearby that people can walk to (keep this to 5-10min at best)